x-pack/plugins/security_solution/public/timelines/components/side_panel/host_details/index.tsx

```diff
@@ -41,19 +41,19 @@ const StyledEuiFlyoutBody = styled(EuiFlyoutBody)`
 `;
 
 const StyledEuiFlexGroup = styled(EuiFlexGroup)`
-  flex: 0;
-`;
-
-const StyledEuiFlexItem = styled(EuiFlexItem)`
-  &.euiFlexItem {
-    flex: 1 0 0;
-    overflow-y: scroll;
-    overflow-x: hidden;
-  }
+  flex: 1 0 auto;
 `;
 
 const StyledEuiFlexButtonWrapper = styled(EuiFlexItem)`
   align-self: flex-start;
+  flex: 1 0 auto;
+`;
+
+const StyledPanelContent = styled.div`
```

Comment:
This fixes (same in network_details): https://github.com/elastic/kibana/issues/92004#issuecomment-782110951